`ScannerTest.testBulkFilesAddChange()` is flaky

**Jetty version(s)**
12.1.x

**Description**
`ScannerTest.testBulkFilesAddChange()` sometimes fails with:

```
java.lang.AssertionError: 
Expected: iterable with items ["/home/lorban/work/webtide/jetty/jetty-12.1.x/jetty-core/jetty-util/target/tests/oeju.ScannerTest/testBulkFilesAddChange/A0"] in any order
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.util.ScannerTest.testBulkFilesAddChange(ScannerTest.java:571)
```

See: https://jenkins.webtide.net/blue/organizations/jenkins/jetty.project/detail/PR-14437/8/tests

--- a/jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/ScannerTest.java
+++ b/jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/ScannerTest.java
@@ -744,7 +744,7 @@ public class ScannerTest
 
     private static void touch(Path file) throws IOException
     {
-        touch(file, (ft) -> System.currentTimeMillis());
+        touch(file, (ft) -> ft + 1L);
     }
 
     private static void touch(Path file, Function<Long, Long> fileTimeDelta) throws IOException

